hypertrace-graphql-labels-schema/src/main/java/org/hypertrace/graphql/label/dao/LabelDao.java

```diff
@@ -2,9 +2,18 @@
 
 import io.reactivex.rxjava3.core.Single;
 import org.hypertrace.core.graphql.common.request.ContextualRequest;
+import org.hypertrace.graphql.label.request.LabelCreateRequest;
+import org.hypertrace.graphql.label.request.LabelDeleteRequest;
+import org.hypertrace.graphql.label.request.LabelUpdateRequest;
+import org.hypertrace.graphql.label.schema.Label;
 import org.hypertrace.graphql.label.schema.LabelResultSet;
 
 public interface LabelDao {
-
   Single<LabelResultSet> getLabels(ContextualRequest request);
+
+  Single<Label> createLabel(LabelCreateRequest request);
+
+  Single<Boolean> deleteLabel(LabelDeleteRequest request);
```

Comment:
So what does it mean to delete a label? Does this delete it across the system anywhere it may apply?
If yes: that's going to be difficult to implement, since this service won't and shouldn't know all those places.
If no: we're going to get dangling references with IDs pointing back to a label that doesn't exist

My suggestion: let's remove delete until we understand what its semantics are meant to be. We can add it without it affecting any of the existing code.